src/services/DateService.ts

```diff
@@ -45,8 +45,8 @@ export class DateService {
       relativeTime: {
         future: "in %s",
         past:   "%s ago",
-        s:  "%dsec",
-        ss: "%dsec",
+        s:  "<1min",
```

Comment:
Please add spaces to all of these between the number and the unit.  "m" should be "mo"

Reply:
done